x509util/certcheck/certcheck.go

```diff
@@ -95,6 +104,39 @@ func main() {
 	}
 }
 
+func chainFromSite(target string) ([]*x509.Certificate, error) {
+	u, err := url.Parse(target)
+	if err != nil {
+		return nil, fmt.Errorf("%s: failed to parse URL: %v", target, err)
+	}
+	if u.Scheme != "https" {
+		return nil, fmt.Errorf("%s: non-https URL provided", target)
+	}
+	host := u.Host
+	if !strings.Contains(host, ":") {
+		host += ":443"
```

Comment:
does `tls.Dial` not do the right thing by default, or is this about begin explicit for logging etc.?

Reply:
d'oh, thanks